config/config.go

```diff
@@ -176,9 +176,10 @@ func (u URL) MarshalYAML() (interface{}, error) {
 
 // Config is the top-level configuration for Prometheus's config files.
 type Config struct {
-	GlobalConfig  GlobalConfig    `yaml:"global"`
-	RuleFiles     []string        `yaml:"rule_files,omitempty"`
-	ScrapeConfigs []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
+	GlobalConfig        GlobalConfig     `yaml:"global"`
+	AlertRelabelConfigs []*RelabelConfig `yaml:"alert_relabel_configs,omitempty"`
```

Comment:
This is as we discussed.

But this is a good time to discuss opening a new top-level section called `alerts`. 
@brian-brazil, we need a section to handle service discovery for Alertmanagers. So I'd imagine we'd end up somewhere like:


```yaml
alerts:
  relabel_configs:
  - ...
  - ...
  alertmanagers:
    kubernetes_sd_config:
      ...
    relabel_configs:
    -  ...
```

I think we can save ourselves making lists of SDs in this case as well and should probably only allow using a single one, to be consistent.

(Why did we suffix everything with `_config` again in the past? Seems like bloat at best.)